### PR TITLE
Handling assets attribute with webpack

### LINKED ISSUE #80: feat: Asset handling via Frontmatter
Basically https://github.com/vuejs/vuepress/issues/79. Common feature in Hexo and the likes, where you can set assets like a post thumbnail in the frontmatter:

```yaml
---
image: ./thumbnail.png
---
```

### PR DESCRIPTION
**Pull request by egoist**
**Summary**

Closes #80 

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI/CSS related code, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: packages/saber/lib/utils/assetsAttribute.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// @ts-check
+
+/**
+ * Check if it's external resource
+ * @param {string} str
+ */
+const isExternal = str => /^https?:\/\//.test(str) || /^\//.test(str)
+
+const MARK = '@@!!SABER_ASSET_MARK_e5968b9a!!@@'
```
Comment (krmax44) [inline packages/saber/lib/utils/assetsAttribute.js]: Is `e5968b9a` there just to make it more random?

File: packages/saber/lib/utils/assetsAttribute.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  return result
+}
+
+/**
+ * Replace strings starting with the MARK to `require` call
+ * @param {string} str
+ */
+const markToRequire = str =>
```
Comment (krmax44) [inline packages/saber/lib/utils/assetsAttribute.js]: I think `markToRequire` is a description that fits to both functions, I'd say it even fits `prefixAssets` better. Would prefer something like `requireAssets` or `injectAssets`

File: packages/saber/lib/utils/assetsAttribute.js (hunk 1 above)
Comment (egoist) [reply packages/saber/lib/utils/assetsAttribute.js]: Yep 😂

Comment (krmax44) [reply packages/saber/lib/utils/assetsAttribute.js]: So there's still a chance I won't break anything even if I name my assets `@@!!SABER_ASSET_MAR_!!@@cute-doggo.jpg`, that's awesome :laughing:

**Pull request merged**